SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/GridBasedLBStrategy/GridBasedLBStrategyTest.cpp

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+#include "GridBasedLBStrategy.h"
+#include "TestGridBasedLBStrategy.h"
+#include "CoreMinimal.h"
+#include "AutomationEditorCommon.h"
+#include "Engine/World.h"
+#include "GameFramework/GameStateBase.h"
+#include "AutomationCommon.h"
+#include "GameFramework/DefaultPawn.h"
+
+#define GRIDBASEDLBSTRATEGY_TEST(TestName) \
+	TEST(Core, UGridBasedLBStrategy, TestName)
+
+UWorld* TestWorld;
+TMap<FName, AActor*> TestActors;
+UGridBasedLBStrategy* Strat;
+
+UWorld* GetAnyGameWorld()
+{
+	UWorld* World = nullptr;
+	const TIndirectArray<FWorldContext>& WorldContexts = GEngine->GetWorldContexts();
+	for (const FWorldContext& Context : WorldContexts)
+	{
+		if (((Context.WorldType == EWorldType::PIE) || (Context.WorldType == EWorldType::Game)) && (Context.World() != NULL))
```

Comment:
Nit - might be easier to read without the extra brackets and split to two lines.
Also NULL -> nullptr

Reply:
Done